js/index.js

```diff
@@ -0,0 +1,41 @@
+$('document').ready(function(){
+
+	console.log('ready');
+
+	$('#submit-btn').click(hitSubmit);
+
+
+	function hitSubmit(event) {
+		event.preventDefault();
+
+		var userInput = $('#city-type').val().trim().toLowerCase();
+
+
+			if (userInput ==='nyc'||userInput ==='new york city'
+				||userInput ==='new york'){
+						$('body').removeClass('sf la austin sydney').addClass('nyc');
+						$('#city-type').val('');
+
+			} else if (userInput ==='san francisco'||userInput ==='sf'
+				||userInput ==='bay area'){
+						$('body').removeClass('nyc la austin sydney').addClass('sf');
+						$('#city-type').val('');
+
+			} else if (userInput ==='los angeles'||userInput  ==='la'
+				||userInput ==='lax'){
+						$('body').removeClass('nyc sf austin sydney').addClass('la');
+						$('#city-type').val('');
+
+			} else if (userInput ==='austin'||userInput ==='atx'){
+						$('body').removeClass('nyc sf la sydney').addClass('austin');
+						$('#city-type').val('');
+
+			} else if (userInput ==='sydney'||userInput ==='syd'){
+						$('body').removeClass('nyc sf la austin').addClass('sydney');
+						$('#city-type').val('');
```

Comment:
Nice job with the form clean up `$('#city-type').val('');` 👍 

Since in all of the `if` `else if` statements you're doing the same thing, you can just move that line of code below all the `if` `else if` blocks. That way you don't have to repeat yourself.

```js
if (userInput ==='nyc'||userInput ==='new york city'||userInput ==='new york'){
  // just do the stuff that's specific to NYC here
  $('body').removeClass('sf la austin sydney').addClass('nyc');
} else if (userInput ==='los angeles'||userInput  ==='la' ||userInput ==='lax'){
  // just the stuff that's specific to LA here
  $('body').removeClass('nyc sf austin sydney').addClass('la');
} // ...

// And now that we're outside the if/else ifs, do the stuff that everyone needs to do
$('#city-type').val('');
```